pages/common/psql.md

```diff
@@ -2,22 +2,22 @@
 
 > PostgreSQL command-line client.
 
-- Connect to *database*. It connects to localhost using default port *5432* with default user:
+- Connect to database. It connects to localhost using default port 5432 with default user:
 
 `psql {{database}}`
 
-- Connect to *database* on given server *host* running on given *port* with *username* given, no password prompt:
+- Connect to database on given server host running on given port with username given, no password prompt:
```

Comment:
"username given" --> "given username" (more natural, and matches the other arguments to this command).

Also, "no password prompt" --> "without a password prompt".